### PR TITLE
Add ANSSI minimal profile to SLE12

### PR DESCRIPTION
**Pull request by teacup-on-rockingchair**
#### Description:

- Add ANSSI minimal profile definition for SLE12 and relevant changes regarding CCE ids and platform compatability

#### Rationale:
- Add ANSSI minimal profile definition for SLE12
- Add sle12 CCEs for ANSSI profile rules
- Add missing cce test for SLE platforms
- Add SLE support for ensure_gpgcheck_never_disabled remediations

### CODE REVIEW

**Review by Mab879 — CHANGES_REQUESTED** [review]
Please look at the [rules](https://github.com/ComplianceAsCode/content/runs/6451389240?check_suite_focus=true#step:6:517) listed by the tests that are missing CCEs.

Thanks for the PR!

**Comment by teacup-on-rockingchair** [comment]
> Please look at the [rules](https://github.com/ComplianceAsCode/content/runs/6451389240?check_suite_focus=true#step:6:517) listed by the tests that are missing CCEs.
> 
> Thanks for the PR!

My bad on this one. I added it to help my work but since all the failing rules are coming from the CIS profile, which SUSE does not officially support cannot assign CCEs to those. So I have 2 options:
- Drop the tests, and hopefulyy when CIS becomes supported by SUSE we'll re-add that piece of code
- Add some list of profiles paramter to the missing_cce test to  check only for provided profiles, or check all if that one is missing 
Please share your opinion. I guess if the second option might be useful for other developers it makes sense. Otherwise I will keep it only internal for now

**Comment by Mab879** [comment]
> > Please look at the [rules](https://github.com/ComplianceAsCode/content/runs/6451389240?check_suite_focus=true#step:6:517) listed by the tests that are missing CCEs.
> > Thanks for the PR!
> 
> My bad on this one. I added it to help my work but since all the failing rules are coming from the CIS profile, which SUSE does not officially support cannot assign CCEs to those. So I have 2 options:
> 
>     * Drop the tests, and hopefulyy when CIS becomes supported by SUSE we'll re-add that piece of code
> 
>     * Add some list of profiles paramter to the missing_cce test to  check only for provided profiles, or check all if that one is missing
>       Please share your opinion. I guess if the second option might be useful for other developers it makes sense. Otherwise I will keep it only internal for now

I think that list of profiles that use the test would be a good addition, but that should be in another PR for easier review. So for this PR, I would be in favor of just removing the test.

**Comment by teacup-on-rockingchair** [comment]
> > > Please look at the [rules](https://github.com/ComplianceAsCode/content/runs/6451389240?check_suite_focus=true#step:6:517) listed by the tests that are missing CCEs.
> > > Thanks for the PR!
> > 
> > 
> > My bad on this one. I added it to help my work but since all the failing rules are coming from the CIS profile, which SUSE does not officially support cannot assign CCEs to those. So I have 2 options:
> > ```
> > * Drop the tests, and hopefulyy when CIS becomes supported by SUSE we'll re-add that piece of code
> > 
> > * Add some list of profiles paramter to the missing_cce test to  check only for provided profiles, or check all if that one is missing
> >   Please share your opinion. I guess if the second option might be useful for other developers it makes sense. Otherwise I will keep it only internal for now
> > ```
> 
> I think that list of profiles that use the test would be a good addition, but that should be in another PR for easier review. So for this PR, I would be in favor of just removing the test.

Thanks for the feedback totally agree. Dropping that commit from the PR, and soon will push that enhancement.

**Comment by Mab879** [comment]
@teacup-on-rockingchair Looks like there is a merge conflict, once that is fixed I will review this PR.

**Review by Mab879 — CHANGES_REQUESTED** [review]
Please rebase this pull request to remove the merge commit. Other than that the PR looks good.

**Comment by teacup-on-rockingchair** [comment]
Thx @yuumasato , not sure how I achieved that :) but now looks more straightforward.

**Review by yuumasato — COMMENTED** [review]
The changes look good to me.

But the PR is missing the profile file, my fault when suggesting the rebase command.
The profile file added in https://github.com/ComplianceAsCode/content/commit/b995c1c9811bd6df5ba432a6aee2024e75eb39e5 is missing.
That file is also using `anssi` instead of `anssi_sle`.

File: products/sle12/profiles/anssi_bp28_minmal.profile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+documentation_complete: true
```
Comment (yuumasato) [inline products/sle12/profiles/anssi_bp28_minmal.profile:1]: @teacup-on-rockingchair Is the intention to name the profile 'minmal`?
I see that `sle15` also got a profile named like that: `anssi_bp28_minmal`

Comment (teacup-on-rockingchair) [reply products/sle12/profiles/anssi_bp28_minmal.profile:1]: This intention was to follow the scheme for rhel8, and once we have all the rules covered will add also high, enhanced and intermediate.

Comment (yuumasato) [reply products/sle12/profiles/anssi_bp28_minmal.profile:1]: Anyway, I can merge this as is, and the typo can be corrected later.

**Review by yuumasato — APPROVED** [review]

**Pull request merged**